.github/workflows/e2e-test.yml

```diff
@@ -19,57 +19,59 @@ jobs:
         working-directory: './e2e'
 
     steps:
-    - name: Checkout Repo
```

Comment:
can you point out what actually changed? as A newcomer to this it's a little hard to follow

Reply:
Actually it looks like it's just the release tracker url?

Reply:
Sorry, everything is in the diff because of applying auto formatting that changed all the indents. I think it is just adding the release tracker url. Most of the other changes are in the script itself.